src/screens/MainApp/DiagnoseRequest/screens/ImageReview/index.js

```diff
@@ -122,18 +109,25 @@ const ImageVisualization = ({ navigation }) => {
     setImageListComponent(buildImageList())
   }, [activeIndex, images])
 
+  useEffect(() => {
+    const handler = BackHandler.addEventListener(
+      'hardwareBackPress',
+      () => {
+        getMorePhotos()
+      }
+    )
+    return () => {
+      handler.remove()
+    }
```

Comment:
Just a thought, should we able to just return `handler.remove`?

Reply:
If `remove` is a method, you'd be passing the function only and it'd be called without the `this` keyword attached to `handler`.
```javascript
const remove = handler.remove
const boundRemove = handler.remove.bind(remove)

remove() // can fail because of "this" not being bound
handler.remove() // will succeed
boundRemove() // will succeed
```